stl/inc/ranges

```diff
@@ -4000,10 +4053,26 @@ namespace ranges {
         };
 
         template <bool _Const>
-        class _Iterator : public _Category_base<_Const> {
+        class _Iterator_base : public _Category_base<_Const> {};
+
+        template <bool _Const>
+            requires forward_range<_Vw>

```

Comment:
Similarly, should this say `_Maybe_const<_Const, _Vw>` based on how \[range.join.with.iterator\] says `present only if Base models forward_range`?